scss/elements/_lists.scss

```diff
@@ -19,7 +19,9 @@
 ul,
 ol {
   list-style-type: none;
-  margin: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+  margin-left: .25rem;
```

Comment:
Nope, I was looking at only radio buttons and check boxes here. Thanks for pointing this out though so I'll know where to look in the future!